### PR TITLE
Sets (most) xenos back to old statistics, with certain castes being buffed.

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
TL;DR - Reverts to pre-kurt stats for most things, with certain castes being buffed if I felt they needed the buff in comparison to their old state.
A lot of frontliners gained .85 sunder mult (Gorger/Warrior/Defender) and a decent chunk of them got around ~30ish more HP.
I kept some of the Kurt buffs if I felt they fit. Mostly plasma gain and max changes on a caste per caste basis.
All rulers now have .8 Sunder resist. In practice this means Shrike has it now.
## Why It's Good For The Game
I'd say the Kurt Xeno experiment broke the balance of the game in a bad way, a lot of the castes that ended up being gigabuffed *did not need it.* Castes like Warrior, Carrier, Crusher got huge buffs which they didn't deserve which made them basically broken while ironically underplayed castes ended up with basically next to no changes.
This still didn't fix any of the core design issues we need to tackle anyway like curbing down emplacements (hi me) making droppods less horrific against map flow, and making vehicles less absurdly lethal and basically uninteractable bricks of unstunnable meat in a game built around displacements, movement, and stuns.

On a side note... PR also basically lied, the vast majority of the cast had no speed changes to note of. The only one I saw was shrike losing .2 speed. Like two others lost .1 speed. This was barely a change at all.
Ideally the Sunder multiplier should make the buffed castes more noticeably tougher and able to stay in the fight longer.

If you have anything to add, please say so in the comments, I do feel like a lot of the Pre-Kurt castes needed some buffs and these are mine.
## Changelog
:cl:
balance: Reverted the vast majority of the Kuro020 Xeno Balance changes. Generally Xenos will be easier to kill. Most of the plasma changes are kept. Read PR #16522
/:cl:

### CODE REVIEW

**Comment by blackdav123** [comment]
tldr: defiler and defender both buffed, zero other changes

**Comment by SauceCat9** [comment]
PEAAAAAAAAAAAAK. Please kills pods and tadpole next.

**Comment by FunkyMonke47** [comment]
yeah, this does not solve the actual issue that pr was originally made for, which is that playing xeno was complete cbt (and there is no one coding to actual fix that issue). This just returns to that status quo. I think there is room to revert some of the buffs, like even partially(like warrior, crusher, and some other stuff like that), but reverting it entirely is lol territory. (kuro is the person who has to approve balance changes, what do you expect to happen with this pr)

**Comment by SimplySmiley123** [comment]
> PEAAAAAAAAAAAAK. Please kills pods and tadpole next.

Kill stuns next.

**Comment by DrAmazing343** [comment]
yeah, no, this feels very very much like a bandaid "fix" that'll come back to once again bite us in the ass once experienced statics resurface or prove themselves and marines establish their chokehold again. I know it's not ided- playing marine is a bit ass if there's a bunch of PVT/PFC chumps around, but I'd rather we focus and attack specific problems instead of reverting to where we were. As Funky says, playing Xeno before was just kinda CBT, and I don't really want that back.

**Comment by Kuro020** [comment]
Not a single one of the problems xenos had that i highlighted in the original pr have been fixed yet.

They have not gotten more content or more options, and still are way behind marines in that regard.

**Comment by Looking4bros** [comment]
Hehehehhe, would be funny to have a Do Not Merge PR that's just a revert of #4434 and  #4436 with #13615 in mind. Maybe reduce all stuns down to 0.1 seconds too. just for shits and giggles. Would be along the lines of a 10% marine armor buff with a 70-90% Xeno slash buff.
